anyway/views/schools/api.py

```diff
@@ -58,8 +58,8 @@ def schools_description_api():
             ),
             not_(
                 and_(
-                    SchoolWithDescription2020.latitude == None,
-                    SchoolWithDescription2020.longitude == None,
+                    SchoolWithDescription2020.latitude is not None,
+                    SchoolWithDescription2020.longitude is not None,
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969